### PR TITLE
Update revisions of RadeonOpenCompute and Khronos ICD

### PR DESCRIPTION
**Pull request by wkwchau**
Wilkin wants to merge 1 commit into RadeonOpenCompute:roc-2.4.x from RadeonOpenCompute:roc-2.4.x-man-update

### CODE REVIEW

File: opencl.xml [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-  <default remote="RadeonOpenCompute" revision="master" sync-c="true" sync-j="4"/>
+  <default remote="RadeonOpenCompute" revision="refs/tags/roc-2.4.0" sync-c="true" sync-j="4"/>
 
   <project path="opencl" name="ROCm-OpenCL-Runtime"/>
   <project path="opencl/compiler/driver" name="ROCm-OpenCL-Driver"/>
-  <project path="opencl/compiler/llvm" name="llvm" revision="amd-common"/>
-  <project path="opencl/compiler/llvm/tools/clang" name="clang" revision="amd-common"/>
-  <project path="opencl/compiler/llvm/tools/lld" name="lld" revision="amd-common"/>
+  <project path="opencl/compiler/llvm" name="llvm" revision="refs/tags/roc-ocl-2.4.0"/>
+  <project path="opencl/compiler/llvm/tools/clang" name="clang" revision="refs/tags/roc-ocl-2.4.0"/>
```
Comment (kzhuravl) [inline opencl.xml]: refs/tags/roc-2.4.0

Comment (wkwchau) [reply opencl.xml]: Done.

File: opencl.xml [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 
-  <default remote="RadeonOpenCompute" revision="master" sync-c="true" sync-j="4"/>
+  <default remote="RadeonOpenCompute" revision="refs/tags/roc-2.4.0" sync-c="true" sync-j="4"/>
 
   <project path="opencl" name="ROCm-OpenCL-Runtime"/>
   <project path="opencl/compiler/driver" name="ROCm-OpenCL-Driver"/>
-  <project path="opencl/compiler/llvm" name="llvm" revision="amd-common"/>
-  <project path="opencl/compiler/llvm/tools/clang" name="clang" revision="amd-common"/>
-  <project path="opencl/compiler/llvm/tools/lld" name="lld" revision="amd-common"/>
+  <project path="opencl/compiler/llvm" name="llvm" revision="refs/tags/roc-2.4.0"/>
```
Comment (kzhuravl) [inline opencl.xml]: refs/tags/roc-ocl-2.4.0 should only be used for llvm and lld, nothing else.

Comment (wkwchau) [reply opencl.xml]: Revert the change of llvm revision.

**Pull request merged**